cosmwasm/contract.ts

```diff
@@ -121,24 +101,55 @@ const registerItsChain = async (
             },
         };
     });
+};
 
-    if (options.update) {
-        for (let i = 0; i < options.chains.length; i++) {
-            const chain = options.chains[i];
-            await validateItsChainChange(client, config, chain, chains[i]);
-        }
+const registerItsChain = async (
+    client: ClientManager,
+    config: ConfigManager,
+    options: ContractCommandOptions,
+    args?: string[],
+    fee?: string | StdFee,
+): Promise<void> => {
+    if (!args || args.length === 0) {
+        throw new Error('At least one chain is required');
+    }
+
+    const chains = buildItsHubChains(config, args);
```

Comment:
```suggestion
    args: string[],
    fee?: string | StdFee,
): Promise<void> => {
    const chains = buildItsHubChains(config, args);
```
bad practice to add this kind of validation everywhere, leads to a lot of superfluous logic. make the code correct by default, in this case by always passing in an `args`